src/pytest_cov/plugin.py

```diff
@@ -226,6 +244,13 @@ def pytest_runtestloop(self, session):
                 compat_session.testsfailed += 1
 
     def pytest_terminal_summary(self, terminalreporter):
+        if self._disabled:
+            markup = {'red': True, 'bold': True}
+            msg = (
+                'WARNING: Coverage disabled by user!'
```

Comment:
Like I said. Your request is valid but my thought process is valid too. That's all I wanted to point out :)